python/lsst/pipe/tasks/postprocess.py

```diff
@@ -455,12 +455,13 @@ class PostprocessAnalysis(object):
     _defaultFuncs = (('coord_ra', RAColumn()),
                      ('coord_dec', DecColumn()))
 
-    def __init__(self, parq, functors, filt=None, flags=None, refFlags=None):
+    def __init__(self, parq, functors, filt=None, flags=None, refFlags=None, forcedFlags=None):
```

Comment:
The `forcedFlags` argument is not documented yet. Similarly, in line 605 it should be mentioned that `forcedFlags` will also be expanded per band.